packages/block-library/src/image/edit.js

```diff
@@ -452,7 +452,7 @@ export function ImageEdit( {
 					context={ context }
 					clientId={ clientId }
 					blockEditingMode={ blockEditingMode }
-					parentLayoutType={ parentLayout?.type }
+					parentLayoutType={ layoutType }
```

Comment:
This isn’t required for the fix and affects nothing at present. It just makes sense to pass the resolved value since otherwise the `Image` component would be getting `undefined` when inside a Gallery. It doesn’t matter, for now, in that particular case because everything it might affect is subordinate to `allowResize` that’s provided through context.